app/views.py

```diff
@@ -30,14 +30,14 @@ def isLoggedIn():
     return True
 
 
-def getUser():
+def getUser(username):
     if not isLoggedIn():
         return {'error': "Not logged in."}
 
     ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
                            test=app.config['TESTING'],
                            token=session['token'])
-    user = ts.get_users(username=session['username'])
+    user = ts.get_users(username=username)
 
     return user
```

Comment:
I think that you should check for errors before returning the user object. Otherwise, we might get some unexpected errors when other parts of the code try to access the user object.